pkg/billing/azureblobstoragebilling_test.go

```diff
@@ -0,0 +1,189 @@
+package billing
```

Comment:
Should this be `billing_test`?

Reply:
that would hide the unexported attributes like getClient from the test package. I tend to not use different packages for testing. IIUC golang does the optimisation of not including testing dependencies unless they are explicitly used in non-test code.

Reply:
> that would hide the unexported attributes like config.getClient from the billing package

Hm, I think these can be worked around with not too much effort, but it can be a follow-up PR.

> IIUC golang does the optimisation of not including testing dependencies unless they are explicitly used in non-test code.

I thought that was only the case for `_test` packages? if that's the case though, that's quite fancy!

Reply:
Just read what you linked here: https://neondb.slack.com/archives/C04DTS134H0/p1720027537887639

Neat!

Reply:
> the customer reported that renaming all the (non-acceptance) test files in go-mysql-server to end in _test.go removed the testify symbols from his binary

yeah, seems like that.